src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -11783,5 +11784,29 @@ void TestMethod()
             await VerifyItemIsAbsentAsync(source, "X");
             await VerifyItemIsAbsentAsync(source, "Y");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(59714, "https://github.com/dotnet/roslyn/issues/59714")]
+        public async Task InvalidConditionalMemberAccessShouldNotThrowExceptions()
+        {
+            var source = @"
+class MyClass
+{
+    public MyClass? MyObject { get; set; }
+    public object? Value<T>() => null;
+
+    public static void F()
+    {
+        var m = new MyClass();
+        m.MyObject?.$$ Value<bool>!();
+    }
+}";
+            List<Exception> exceptions = new();
+            FatalError.Handler = (e, _, _) => exceptions.Add(e);
+
+            await VerifyItemIsAbsentAsync(source, "Value");
+
+            AssertEx.Empty(exceptions, $"{exceptions.Count} exceptions encountered");
```

Comment:
i don't think we need tests that do this.  our system is broken if awaiting any of our product code doesn't cause any exceptions encountered while running to bubble up.

Reply:
I'm not sure how the error handler is supposed to be hooked up in tests, but it's definitely being swallowed in this case
https://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/Providers/AbstractSymbolCompletionProvider.cs,270

Reply:
in tests, FatalError is supposed to kill the test run i thought.  @jasonmalinowski ?

Reply:
This would fix it
https://github.com/dotnet/roslyn/pull/36565